.github/dependabot.yml

```diff
@@ -137,6 +137,30 @@ updates:
     schedule:
       interval: "weekly"
 
+  - package-ecosystem: "npm"
+    directory: "/packages/webhook"
+    groups:
+      slack:
+        patterns:
+          - "@slack/*"
+      dev-sinon:
+        dependency-type: "development"
+        patterns:
+          - "@types/sinon"
+          - "sinon"
```

Comment:
```suggestion
```

🪓 We might've found a way to test without this in the more recent changes:

https://github.com/slackapi/node-slack-sdk/blob/d83f4809d6c1c51577222f2ad83259674d7b1c35/packages/webhook/package.json